cmds/bmp/login_command_test.go

```diff
@@ -72,7 +72,7 @@ var _ = Describe("login command", func() {
 
 	Describe("#Usage", func() {
 		It("returns the usage text of a LoginCommand", func() {
-			Expect(cmd.Usage()).To(Equal("bmp login --username[-u] <username> --password[-p] <password"))
+			Expect(cmd.Usage()).To(Equal("bmp login --username[-u] <username> --password[-p] <password>"))
 		})
 	})
```

Comment:
these login changes should be moved to their own PR. Let's not mix things, unless there is an explicit dependency(ies)